New package: Dionysos v0.0.1

JuliaRegistrator opened the pull request:
- Registering package: Dionysos
- Repository: https://github.com/dionysos-dev/Dionysos.jl
- Created by: @adrienbanse
- Version: v0.0.1
- Commit: 9e9a3a0af48b581686fe95e7b36a778b7aad26a1
- Git reference: v0.0.2

adrienbanse commented:
@blegat Which licence should I put?

goerz commented:
[noblock] The MIT license is the one most commonly used for Julia project, and thus would make your library the most compatible with the rest of the ecosystem

adrienbanse commented:
@JuliaRegistrator register()

JuliaHubRegistrator committed:
New package: Dionysos v0.0.1

UUID: d92c97cf-b87d-42c1-a9c0-25df00b4d958
Repo: https://github.com/dionysos-dev/Dionysos.jl.git
Tree: 041bf5afb647d7e54b917e6b1e580a856085ce8d

Registrator tree SHA: 7242ef64be5953fac9bf8b3efa3a7c4d2d44ca09

adrienbanse commented:
Close

goerz commented:
Close in favor of https://github.com/JuliaRegistries/General/pull/101970
